### DISCUSSION TITLE
Who's Hiring (June 2024)

### DISCUSSION THREAD

**Discussion by manovotny** [vercel/next.js#66658] [Show and tell]
# What is this?

This discussion is for companies looking for people with Next.js experience. Both permanent positions and contract work are welcome.

## Companies

Please use the following format followed by a brief description of the role:

> COMPANY_NAME | ROLE | LOCATION | Full-time | $100-150k base | LINK

Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

### Sample Job Post

Here's a good example of a job post.

> Vercel | Software Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers#positions)
>
> Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our software engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.
>
> I've been at the company for 2 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me directly.

---

If you hire someone through this job board, please [email me](mailto:michael.novotny@vercel.com) and let me know so I can send you some swag. 👍 

If you're a developer looking to get hired, please use #66657 instead.

Several companies have [hired someone](https://twitter.com/nutlope/status/1610667757565083648) as a result!

**Comment by haydenbleasel** [comment]
**Corellium | Senior Design Engineer | Delray Beach, Florida or Remote | Full-time | $180k base | [LINK](https://boards.greenhouse.io/corellium/jobs/5193073004)**

Corellium is a pioneering company in the field of virtualized mobile device environments, providing powerful tools for developers, researchers, and security professionals. Our cutting-edge technology allows for efficient mobile app testing, security research, and development across various platforms. We are dedicated to fostering innovation, ensuring security, and delivering unparalleled solutions to our clients.

We are looking for a Senior Design Engineer to join our team and contribute to three key areas of the company:

- Owning the development of the marketing website, making it an innovative brand ambassador
- Working on the frontend of our application as we work towards redesigning it.
- Special projects with the CPO (me), including using the Vercel AI SDK to make some cool stuff 👌

**Reply by HelixHEX** [reply]
Hi there! Full stack dev with here with 5 years experience and overall 8+ of building products people use. I’d love to chat more about the role and hopefully bring my love for coding and creative mind to the Corellium team. 

P.S check out this cool project I built with Vercels AI SDK to get an idea of my skill set
[myresume.guru](https://myresume.guru)

**Reply by morganfeeney** [reply]
@haydenbleasel what is the salary range?

**Reply by haydenbleasel** [reply]
@morganfeeney whoops sorry, updating now

**Reply by alecwhittington** [reply]
Sent in an app - looks like an interesting opportunity

**Reply by JoseChavez98** [reply]
Hello @haydenbleasel ,

If you are looking for a remote international Frontend Engineer, please [contact me](https://github.com/vercel/next.js/discussions/66657#discussioncomment-9713549).

**Reply by miamiruby** [reply]
Also interested my email phone and linkedin are 561-213-3854 paul@miamiphp.org https://www.linkedin.com/in/paulkruger/

**Comment by akash-w97** [comment]
[CLOSED]

**Ramp | Senior+ Frontend Engineer | US only (New York, Miami or San Francisco preferred) | Full-time | $225K – $275K base | [Link](https://jobs.ashbyhq.com/ramp/54254ba5-d532-416a-9923-28eedb3520f7)**

[Ramp](https://ramp.com/) is the ultimate platform for modern finance teams. Combining corporate cards with expense management, bill payments, vendor management, accounting automation, travel and more, Ramp's all-in-one solution is designed to save businesses time and money, and free finance teams to do the best work of their lives. Our mission is to help build healthier businesses, and it’s working: over 25,000 businesses on Ramp to save an average 5% and close their books 8x faster. 

The Ramp Web Engineering team is 5 people strong and is currently searching for a 6th engineer to help tech lead the iterative migration of our marketing site to Next.js over the upcoming year. Our ideal candidate has a proven track record in building and optimizing enterprise-scale websites, is a great cross-functional communicator, and possesses a knack for getting the visuals right.

If this sounds like the opportunity for you, [please apply at this link](https://jobs.ashbyhq.com/ramp/54254ba5-d532-416a-9923-28eedb3520f7) and feel free to reach out to [me](https://www.linkedin.com/in/akashwadawadigi/) or [Louis](https://www.linkedin.com/in/louis-weis-41a0b090/) on LinkedIn with any further questions :)

**Reply by BenAHammond** [reply]
Looks great! Application sent in.

**Reply by gunnerhowe** [reply]
Application submitted. Very cool coding puzzle. Reminds me of cicada 3301

**Reply by dsosborn** [reply]
Loved the Capture the Flag, nicely done! (But unfortunately I'm here to stay in Chicago)

**Reply by miamiruby** [reply]
Also near miami and interested my email phone and linkedin are 561-213-3854 
[paul@miamiphp.org](mailto:paul@miamiphp.org) 
https://www.linkedin.com/in/paulkruger/

**Comment by nileriver2323** [comment]
San Francisco 49ers | Fullstack Software Developer | Santa Clara, CA | Full-time | $120-130k base | [LINK](https://career4.successfactors.com/sfcareer/jobreqcareerpvt?jobId=6501&company=SF49SP&st=18A90ED6709FB4334BF161BAA2EA7C6BFC2501A5)

The San Francisco 49ers are looking to add a fullstack software developer to their football systems group. We are in the process of building a V2 of our scouting application using a new tech stack of Next.js and FastAPI. The application supports all departments within football (scouting, coaches, player performance, etc.). We are still early on in the conversion and this role would help shape the direction of the technology going forward.

Feel free to reach out to [me](https://www.linkedin.com/in/neil-mayfield-aa898121/) if you have any questions.

**Reply by YousifAbozid** [reply]
I just sent to you on Linkedin, check your inbox, thanks.

**Reply by megha-tnt** [reply]
I added my resume in the mentioned link.Please check

**Reply by laskarDipankar** [reply]
Hi, if you are still looking for a dev.

Location: Assam, India
Remote: Yes
Willing to relocate: No
Technologies:
Frontend: Nextjs, React
Backend: nodejs, fastify
Database: mysql, postgresql, mongodb
AI Tool: Langchain
Resume/CV: https://www.linkedin.com/in/dipankar-laskar/
Website: https://github.com/mundanecoder
Email: [dipankarlaskar.work@gmail.com](mailto:dipankarlaskar.work@gmail.com)
Last Project: Built an AI tool using Langchain
Actively looking for: Fullstack role
Desired Salary Range: 60,000 - 90,000 INR per month (approximately 720 - 1,080 USD per month)

**Comment by Sashkan** [comment]
Station F | Front-end Senior Developer | Paris, France | Full-time | $120-130k base | [LINK](https://jobs.stationf.co/companies/station-f/jobs/developer-front-end-javascript-h-f_paris)

STATION F is the world's biggest startup campus based in Paris and backed by Xavier Niel. Currently 50,000m2, our campus is home to 1000 early-stage startups participating in 30 different startup programs. We have a number of additional facilities and resources, including Flatmates (our coliving), where 600 entrepreneurs currently live.

Feel free to reach out to [me](https://www.linkedin.com/in/potran/) if you have any questions.

**Reply by YousifAbozid** [reply]
I just set to you a connect request on Linkedin, here is my [portfolio](https://yousifabozid.github.io)
Also here is my [resume](https://github.com/YousifAbozid/YousifAbozid/blob/source/README.md)
If you think that I am a good fit for your team, feel free to reach out to me, have a good day.

**Reply by laskarDipankar** [reply]
if you are looking for a fullStack developer with 1+ years of experience ,

Location: Assam, India
Remote: Yes
Willing to relocate: No
Technologies:
Frontend: Nextjs, React
Backend: nodejs, fastify
Database: mysql, postgresql, mongodb
AI Tool: Langchain
Resume/CV: https://www.linkedin.com/in/dipankar-laskar/
Website: https://github.com/mundanecoder
Email: [dipankarlaskar.work@gmail.com](mailto:dipankarlaskar.work@gmail.com)
Last Project: Built an AI tool using Langchain
Actively looking for: Fullstack role
Desired Salary Range: 60,000 - 90,000 INR per month (approximately 720 - 1,080 USD per month)

**Comment by bitttttten** [comment]
**Senior Front-end Developers @ Creative Fabrica in Amsterdam, NL**

We're Creative Fabrica, a fast scaling tech start-up from Amsterdam on the look out for experienced Front-end Developers to work with us creating new AI tools and help build the crafter ecommerce marketplace and community platforms with TypeScript, Next.js and Tailwind with our GraphQL and Golang backend.

You'll be joining small teams where we have a great building culture. We have a lot going on, so you can be a part of the AI tools, design tools, or our large crafting marketplace (think 40m+ visitors a month). Our Generative AI models are trained in house and our AI tools are built in house too! If you're excited about working on new projects and having a real impact in their development then this is the role for you.

TLDR:
🇳🇱 Office in Amsterdam with lunch provided and Friday drinks
👩‍💻 React, Next.js, Turborepo, Tailwind, Vitest, Radix UI, Vite, AWS, Cloudflare
🤖 Work with unique, first to market, AI tools and a popular crafters marketplace

Any other questions let me know! 

(we're not looking for any remote positions, we're looking at hiring locally in our office in the centre of Amsterdam)

Apply here: https://grnh.se/eeb72142teu

**Reply by morganfeeney** [reply]
@bitttttten what's the salary range?

**Comment by rehantoday** [comment]
UXBERT LABS | SENIOR FRONTEND ENGINEER | REMOTE | FULL-TIME | [LINK](https://uxbert.bamboohr.com/careers/159)

We are looking for Senior Frontend Engineer (5+ years exp., React and Next.js, State Management, Build tools, SDKs, RTL-Multi Languages, Personalization)
If you've relevant experience and would like to develop sites which will be used by millions of users, please apply.

**Reply by morganfeeney** [reply]
@rehantoday what's the salary range?

**Reply by miamiruby** [reply]
Interested my email phone and linkedin are 561-213-3854 
[paul@miamiphp.org](mailto:paul@miamiphp.org) 
https://www.linkedin.com/in/paulkruger/

**Comment by RLesser** [comment]
**Nomic AI | Senior Web/Frontend Engineer | NYC In-person | Full-Time | $100-200k Base | [LINK](https://nomic-ai.notion.site/Careers-Nomic-be1323b7fdef45dc994ed918a2c0ff69)** 

[Nomic](https://atlas.nomic.ai/) is building the next generation of data interfaces for exploring and leveraging massive sets of text and images, all in the browser. As an example, here's [1.4 million english wikipedia biographical articles](https://atlas.nomic.ai/data/nomic/wiki-people/map) embedded with [nomic-text-embed-v1.5](https://huggingface.co/nomic-ai/nomic-embed-text-v1.5) and accompanied by tons of structured metadata.

We are looking for a senior web/frontend engineer with significant experience in React and Next.js. Experience with data visualization and data processing in the browser is a big plus. Please [reach out](https://www.linkedin.com/in/robert-lesser-9b925a115) if you have questions!

**Reply by aizaysi** [reply]
I'm interested in your job.
If I wanna work with you, what do i have to do?
waiting for your response.

**Reply by miamiruby** [reply]
Interested my email phone and linkedin are 561-213-3854 
[paul@miamiphp.org](mailto:paul@miamiphp.org) 
https://www.linkedin.com/in/paulkruger/

**Comment by dudeOFprosper** [comment]
CompareCredit | Senior Javascript Engineer | Remote or Hybrid in Charleston SC | Full-Time | $90-120k Base | [LINK](https://www.linkedin.com/jobs/view/3958529979/)

At CompareCredit.com, our mission is to help people make smart and informed financial decisions. We provide a comprehensive analysis of credit card offers from leading lenders across the United States so that consumers can compare them side-by-side and find the best fit for their needs. Our team of experts is dedicated to providing detailed reviews and helpful tips on how to maximize rewards, minimize fees, select low interest rates, and more – all in an effort to empower individuals with the tools they need to take control of their finances. At CompareCredit, there are No Games. Just Comparisons.


About the Role:

Are you an enthusiastic and talented developer eager to tackle complex technical challenges?
We're seeking a Senior JavaScript Engineer to join our team. If you're excited about pushing the boundaries of web development and have a track record of success with modern JavaScript frameworks, we want to hear from you!

**Reply by laskarDipankar** [reply]
Hi there! I'm interested in this developer role. Here's a quick overview:

Location: Assam, India (Remote only)
Tech stack:

Frontend: Nextjs, React
Backend: Node.js, Fastify
Database: MySQL, PostgreSQL, MongoDB
AI: Langchain


Last project: Built an AI tool using Langchain
GitHub: github.com/mundanecoder
LinkedIn: linkedin.com/in/dipankar-laskar/
Desired salary: 60,000 - 90,000 INR/month

If this fits your needs, please reach out at [dipankarlaskar.work@gmail.com](mailto:dipankarlaskar.work@gmail.com). Thanks!

**Reply by miamiruby** [reply]
Interested my email phone and linkedin are 561-213-3854 
[paul@miamiphp.org](mailto:paul@miamiphp.org) 
https://www.linkedin.com/in/paulkruger/

**Reply by Moemen12** [reply]
Interested 
Portfolio : https://saadeh-portfolio.onrender.com
email : moemensaadeh936@gmail.com
https://www.linkedin.com/in/moemen-saadeh-707436260/

**Comment by shu1xt** [comment]
One X Tech | Senior Software Engineer (React/Next/Tailwind) | Remote (Global) | Full-time | Competitive Salary | [LINK](https://www.onextech.com/careers)

One X Tech is actively seeking a Senior Software Engineer specializing in React and TailwindCSS for a fully remote position. Our development team is global, focusing on modern web applications using NextJS, Vercel, Supabase, Typescript, and TailwindCSS. We prioritize candidates who are comfortable with the GMT+8 timezone and have solid experience in Javascript, particularly with React (NextJS) and TailwindCSS.

With a culture of innovation, One X Tech serves leading enterprises by building technological advancements. This role offers the chance to join an energetic and dynamic team dedicated to excellence in modern web application development. If you're passionate about pushing the boundaries of web development and have the skills listed, we would love to hear from you. Apply through the link provided or send your resume directly to [careers@onextech.com](mailto:careers@onextech.com) with **"[Next Jul 2024]: Remote Software Engineer Application"** as the subject line. Join us in creating extraordinary digital experiences.

**Reply by aizaysi** [reply]
I am really interested in  your job and I sent a application email.
I am waiting for your reply.

I don't wanna see this.
![image](https://github.com/user-attachments/assets/e8f55e30-731d-4700-a960-91259b06d360)

**Comment by jonschmidt** [comment]
It's Complicated | Senior Frontend Engineer (React/Next/Tailwind) | Remote (EU preferred) | Full-time | Competitive Salary | [LINK](https://glacier-book-d32.notion.site/Looking-for-a-Senior-Frontend-Engineer-a40afe415e334aaeb99e25e8c5e14e08)

Its Complicated is looking for a Senior Frontend engineer to join us as the second engineer of our small but mighty mental health focused platform. Expect to have a lot of autonomy, responsibility, flexible hours, fast paced feature cycles, and an extremely supportive team around you with excellent senses of humour. Our product team is in Berlin, and we would ideally like someone not too far away. An interest in mental health is a plus but not a requirement.

**We are currently only looking in GMT+0, GMT+1, GMT+2, GMT+3, and we are not offering any paid relocation**

We would love a full timer who wants to stay for a while. Expect a competitive salary range and significant equity package to boot. Check the link for more details and the application email...

**Reply by miamiruby** [reply]
Interested my email phone and linkedin are 561-213-3854 
[paul@miamiphp.org](mailto:paul@miamiphp.org) 
https://www.linkedin.com/in/paulkruger/

**Reply by Moemen12** [reply]
Interested 
Portfolio : https://saadeh-portfolio.onrender.com
email : moemensaadeh936@gmail.com
https://www.linkedin.com/in/moemen-saadeh-707436260/

**Comment by matannahmani** [comment]
Ada AI | Software Engineer | Remote (Asia) | Full-time | Competitive Salary | [Link](https://www.linkedin.com/jobs/view/3937529665)

We are looking for two additional founding engineers at Ada AI (Tokyo/Seoul timezone).

About Us:
We're building the first iteration of ai meeting work companion (real-time).
Our company follows a remote-first approach with competitive salaries.
We are backed by some of the industry titans (Summit Partners, DCVC)

Required Skills:
Typescript (E2E, backend, and frontend)
NextJS and React
Playwright / Puppeteer / automation tools
Relational databases
Passion for low latency

If you're interested and have these skills in your stack, please:
email me directly at: matan@im-ada.ai

**Reply by gitt-er** [reply]
Hi, I am interested in this opportunity. I sent an application.

Thanks

**Reply by miamiruby** [reply]
Interested my email phone and linkedin are 561-213-3854 
[paul@miamiphp.org](mailto:paul@miamiphp.org) 
https://www.linkedin.com/in/paulkruger/

**Comment by DrewFH** [comment]
[Filmhub](filmhub.com) | Lead and Senior Product Engineers| Remote (North America and Europe) | Full-time | $80-170k base (varies by geographic location and experience) | [Link](https://jobs.ashbyhq.com/filmhub/6650f4d0-0d59-4792-ba6b-33a6a3a3a450)

Filmhub is expanding our engineering team and is looking for senior and lead engineers. The Product & Engineering team develops, launches, and maintains our top tech. Developer experience is First at Filmhub, embracing cutting-edge frameworks and tools. We ship often and make deployments fast, which keeps us on our toes. Filmhub engineers have full ownership of what we build from beginning to end — not just a small slice of it. New team members deploy their first feature within their first few days, making a tangible change they own early on and all the time.

Work with Klaus! https://github.com/klausbadelt

React, typescript, javascript, next.js, Vercel, serverless. 

We have a remote-first work policy that allows you to work from anywhere in the world! This opens up incredible opportunities for those who may not have access to film industry jobs locally or are specifically seeking remote work options. Joining our team means collaborating with colleagues from different countries, cultures, and time zones, creating a truly diverse and inclusive work environment.

**Comment by pawlean** [comment]
For those wondering what happened to these posts... we're so back! https://github.com/vercel/next.js/discussions/69888
